<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ex1Dialog</class>
 <widget class="QDialog" name="Ex1Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>翻打</string>
  </property>
  <widget class="Ex1LineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>金额</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>传票编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_current_t_num">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>59</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>累计金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>第1本总计金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>第2本总计金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>第3本总计金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>75</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>考试时间倒计时：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_left">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_current_b_num">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ex1LineEdit</class>
   <extends>QLineEdit</extends>
   <header>ex1lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
